openwisp_monitoring/monitoring/utils.py

```diff
@@ -58,5 +60,16 @@ def create_database():
     databases = [database['name'] for database in items]
     # if database does not exists, create it
     if settings.INFLUXDB_DATABASE not in databases:
-        db.create_database(settings.INFLUXDB_DATABASE)
-        logger.info('Created inlfuxdb database {0}'.format(settings.INFLUXDB_DATABASE))
+        for n in range(5):
+            try:
+                db.create_database(settings.INFLUXDB_DATABASE)
+                break
+            except InfluxDBClientError:
+                warn_and_delay(n)
+        logger.info(f'Created influxdb database {settings.INFLUXDB_DATABASE}'
+
+
+def warn_and_delay(n):
+    print(f'Caught an error, retrying ({n}/5 tries)')
```

Comment:
Use `Got error while connecting to timeseries DB. Retrying again in 3 seconds. Attempt {n} out of 5.`